src/components/CalcButtons/Button.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {Text, TouchableOpacity} from 'react-native';
+
+const Button = ({label, handlePress, style = {}}) => {
+  return (
+    <TouchableOpacity onPress={() => handlePress(label)}>
```

Comment:
Taking a press event and changing it's behaviour is not recommended.

Remember separation of concerns. Button should just render a button and execute a function when it's pressed. What the function does should be defined by the component that uses button, not button itself.